core/src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -18,10 +18,7 @@
 import me.leoko.advancedban.utils.tabcompletion.TabCompleter;
 import org.apache.commons.lang3.ArrayUtils;
 
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.GregorianCalendar;
-import java.util.List;
+import java.util.*;
```

Comment:
Please define the imports required that are relevant for the changes rather than using a wildcard.